### PR TITLE
[TW-1650] Postman Vault March non-Enterprise

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
In this release, Postman won't preserve your vault secrets after signing out of the web app. Previously we documented that the web app was required to preserve secrets on sign out.

* Clarified that you need your vault key to access your Postman Vault, and not your vault secrets. The reason is that the vault key doesn't give you access to your secrets because secrets aren't preserved, and the key is required when you sign in.
* Clarified that secrets aren't preserved when you sign out, and this might be a reason why your secrets are unresolved when you sign in.
* There are now only features that require the desktop app. Updated referenced to the "Feature availability" section to reflect this.
* Updated a screenshot with copy changes.

---

[Beta site](https://tw-1650-postman-vault-march-non-enterprise.learning.postman-beta.com/docs/sending-requests/postman-vault-secrets/)

### CODE REVIEW

**Review by petewitcher-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/sending-requests/postman-vault-secrets.md [hunk 1]
```diff
@@ -41,6 +41,10 @@ Vault secrets are created at the *vault* scope, which is the broadest scope comp
 
 > If a variable in a different scope has the same name as a vault secret, the value stored in a narrower scope won't be used. For example, if there is a collection variable named `postman-api-key` and a vault secret also named `postman-api-key`, the collection value won't be used when the request runs. Learn how to [reference vault secrets](#use-vault-secrets) in Postman.
 
+<!-- -->
+
+> If you're using the Postman web app with Safari as your web browser, it deletes vault secrets from your local instance of Postman after seven days of inactivity. You must use a different web browser if you want your vault secrets available for more than seven days without inactivity. Learn about the [browser requirements](/docs/getting-started/installation/installation-and-updates/#browser-requirements) for the Postman web app.
```
Comment (ravi-prasad07) [inline src/pages/docs/sending-requests/postman-vault-secrets.md:46]: @MatthewPinsker-postman 
"You must use a different web browser if you want your vault secrets available for more than seven days without inactivity" 
This conveys the message that there should be no inactivity(meaning there should be some activity) for us to retain the secrets for more than seven days. isn't that not the intended message?

Comment (MatthewPinsker-postman) [reply src/pages/docs/sending-requests/postman-vault-secrets.md]: Thanks for catching this typo. It's now fixed.

**Review by Suhas-Gaikwad — APPROVED** [review]
LGTM

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Review by petewitcher-postman — COMMENTED** [review]
LGTM!

**Review by petewitcher-postman — APPROVED** [review]
LGTM!

**Pull request merged**